src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const object = {};
+
+  const splittedSource = String(sourceString).split(';');
+
+  splittedSource.forEach((item) => {
+    const [key, value] = item.split(':');
+
+    if (key && value) {
+      object[key.trim()] = value.trim();
```

Comment:
If there are duplicate keys in the input string, the current implementation will overwrite previous values with the same key. Consider whether this behavior is acceptable or if you need to handle duplicates differently.